src/react-imgix.jsx

```diff
@@ -9,6 +9,7 @@ import constructUrl from "./constructUrl";
 import { deprecatePropsHOC, ShouldComponentUpdateHOC } from "./HOCs";
 
 import { warning, shallowEqual, compose, config, CONSTANTS } from "./common";
+import { isGenericTypeAnnotation } from "@babel/types";
```

Comment:
How does this work, exactly? Importing this type annotation affects the output file, even though the imported object is never referenced?

Reply:
Whoops! VS code must have added this automatically when I’ve mistakenly used some suggested option. I’ll remove this and rebase. This slipped passed me. Nice catch ✌️